README.md

```diff
@@ -20,47 +20,61 @@ that you plan to manage!
 
 ## Pre-requisites ##
 
-- You must run these playbooks as a user that has ssh access to each target host
-  in your inventory and sudo privileges on each target host.
-- You must create a simple inventory file containing the name or IP address
-  of each target host that you wish to manage users on (one host per line).
-  A sample inventory file might look like this:
-
-  ```console
-  my_server
-  my_database
-  sample.mydomain.com
-  db.mydomain.com
-  192.168.1.5
-  192.168.1.6
+- You must run these playbooks as a user that has ssh access to and
+  sudo privileges on each target host in your inventory.
+- You must create an Ansible inventory file at the root of this
+  project containing the name or IP address of each target host on
+  which you wish to manage users.  A sample inventory file might look
+  like this:
+
+  ```yaml
+  all:
+    hosts:
+      my_ungrouped_host:
+    children:
+      group1:
+        hosts:
+          my_server:
+          my_database:
+      group2:
+        hosts:
+          sample.mydomain.com:
+          db.mydomain.com:
+      group2:
```

Comment:
I don't think this is supposed to be a duplicate key.

```suggestion
      group3:
```

Reply:
You are correct sir!